packages/djangoflow_app/lib/src/bloc/app_cubit.dart

```diff
@@ -23,11 +23,9 @@ class AppState with _$AppState {
 }
 
 class AppCubit extends HydratedCubit<AppState> {
-  static AppCubit get instance => _instance;
```

Comment:
Hi, can you please explain why do we need to change the singleton pattern here?

Reply:
This makes it hard to test this class because the Cubit stream is closed by the first `blocTest` method once it is called. I found a new solution which is to add a new method that returns a new instance of the class for testing purposes.

Reply:
@visibleForTesting Annotation can help us to provide the mockable class to override the default AppCubit instance in that case.
 
 ```dart
   @visibleForTesting
  static set value(AppCubit appCubit) => _instance = appCubit; 
 ```

Reply:
If you look at the latest commit I already added this block of code for testing purposes which is similar to what you have above
`  @visibleForTesting
  static AppCubit getNewInstance() => AppCubit._internal();`

Reply:
I see, sounds good to me.